src/redux/explorer.ts

```diff
@@ -439,23 +439,6 @@ export const explorerInit = () => async (
   });
 };
 
-/**
- * Emits a portfolio request. The result is handled by a listener in
- * `listenOnAddressMessages`.
- *
- * @param address The address.
- * @param currency The currency to use.
- */
-export const emitPortfolioRequest = (address: string, currency?: string) => (
-  _: Dispatch,
-  getState: AppGetState
-) => {
-  const nativeCurrency = currency || getState().settings.nativeCurrency;
-  const { addressSocket } = getState().explorer;
-
-  addressSocket?.emit(...portfolioSubscription(address, nativeCurrency));
```

Comment:
u can remove `portfolioSubscription`

Reply:
actually seems like there might be some stuff to be removed in `listenOnAddressMessages` as well